src/java/org/apache/cassandra/db/ArrayClustering.java

```diff
@@ -32,7 +32,7 @@ public ArrayClustering(byte[]... values)
 
     public long unsharedHeapSize()
     {
-        if (this == ByteArrayAccessor.factory.clustering() || this == ByteArrayAccessor.factory.staticClustering())
+        if (this == ByteArrayAccessor.factory.clustering())
```

Comment:
I assume this change is because of the removal of STATIC_CLUSTERING from ByteArrayObjectFactory?

Reply:
Yes, a byte array clustering can no longer be static (and the reason for that is that using a separate byte array static clustering causes serious problems because a lot of code compares with `STATIC_CLUSTERING` directly).